src/Runtime/Gecko_ExceptionPPC.c

```diff
@@ -0,0 +1,44 @@
+#include <Runtime/Gecko_ExceptionPPC.h>
+
+typedef struct ProcessInfo {
+    __eti_init_info* exception_info;
+    char* TOC;
+    int active;
+} ProcessInfo;
+
+static ProcessInfo fragment_info[1];
+
+#ifndef NON_MATCHING
+// unused asm function to trigger peephole off bug
+static asm void unused()
+{
+}
+#endif
```

Comment:
This works differently than just `#pragma peephole off`?

Reply:
No it works the same, just felt it lines up with the original source better than a bare pragma.